### ISSUE TITLE
[Average Price] 'NaN' when less than  5 SRP results

### ISSUE THREAD

**Issue by luqven** [luqven/truePrice#4]
``` javascript
{... prices: [ '$265.00', '$267.00', '$278.00' ],   averagePrice: 'n/a'}
```

A prices array with only 3 results makes it so `calcAveragePrice()` returns `NaN`

**Comment by luqven** [comment]
Since we're getting an `averagePrice` of `n/a` we can narrow down the error to the [`calcAveragePrice()`](https://github.com/luqven/truePrice/blob/14fb2472ac1cc189549e27c3f840219c7662e08d/scraper/index.js#L155-L160) function.

Most likely, issue stems from how we're calculating [IQRs](https://github.com/luqven/truePrice/blob/14fb2472ac1cc189549e27c3f840219c7662e08d/scraper/index.js#L126-L129) for smaller arrays.

**Comment by luqven** [comment]
Something like the changes bellow should fix this error

```javascript
    // if < 4 values in arr, find average between first and last and set as q1
    if (values.length < 4) { q1 = (values[0] + values[-1] / 2); }
    else { q1 = values[Math.floor(values.length / 4)]; }
    // Likewise for q3.
    const q3 = values[Math.ceil(values.length * (3 / 4))];
    const iqr = q3 - q1;
```

**Comment by luqven** [comment]
Addressed by #5

**Comment by luqven** [comment]
#5 does not apparently fix this... investigating
